### PR TITLE
initial version of the sample

### PR DESCRIPTION
**Pull request by rberrelleza**

### CODE REVIEW

File: okteto.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: hello-world
+image: mcr.microsoft.com/dotnet/core/sdk
+environment:
+- ASPNETCORE_ENVIRONMENT=Development
+command:
+- bash
+workdir: /usr/src/app
```
Comment (pchico83) [inline okteto.yml:7]: can we remove `workdir` and use the default?

**Review by pchico83 — APPROVED** [review]

**Comment by pchico83** [comment]
@rberrelleza is it possible to use a debugger?

**Comment by rberrelleza** [comment]
> @rberrelleza is it possible to use a debugger?

I think that's better for the vscode integration. Need to dustoff my VS skills 🤣

File: okteto.yml (hunk 1 above)
Comment (rberrelleza) [reply okteto.yml:7]: That's the workdir of the dev image, I want to keep it for parity

**Pull request merged**